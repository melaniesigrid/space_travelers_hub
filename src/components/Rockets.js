import React from 'react';
import { useSelector } from 'react-redux';
import { Container } from 'react-bootstrap';
import RocketCard from './RocketCard';

const Rockets = () => {
  const myRockets = useSelector((state) => state.rockets);
  return (
    <Container>
      <br />
      {
        myRockets.map((rocket) => (
          <RocketCard
            key={rocket.id}
            id={rocket.id}
            rocketName={rocket.rocketName}
            description={rocket.description}
            flickrImages={rocket.flickrImages[0]}
          />
        ))
      }
    </Container>
  );
};

export default Rockets;
