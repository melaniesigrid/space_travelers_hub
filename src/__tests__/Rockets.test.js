import renderer from 'react-test-renderer';
import '@testing-library/jest-dom';
import { Provider } from 'react-redux';
import store from '../redux/configureStore';
import Rockets from '../components/Rockets';

it('Rockets Component renders correctly', () => {
  const tree = renderer
    .create(
      <Provider store={store}>
        <Rockets />
      </Provider>,
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
