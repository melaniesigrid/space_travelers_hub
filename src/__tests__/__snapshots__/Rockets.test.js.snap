// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Rockets Component renders correctly 1`] = `
<div
  className="container"
>
  <br />
</div>
`;
